pages/public_cloud/public_cloud_network_services/tutorial-stormshield_network_security_vrack/guide.fr-fr.md

```diff
@@ -0,0 +1,560 @@
+
+Dans cette étape, nous configurons le vRack, un réseau virtuel privé fourni par OVHcloud. Le vRack vous permet d'interconnecter plusieurs instances ou serveurs au sein d'un environnement Public Cloud, assurant ainsi l'isolation du réseau tout en maintenant une communication sécurisée. En ajoutant votre projet Public Cloud et votre bloc Additional IP au même vRack, vous pouvez permettre à vos instances SNS de communiquer de manière sécurisée, tout en gardant le contrôle total de la gestion des adresses IP. Le réseau privé vRack vous permet également de sécuriser des serveurs Bare Metal Cloud ou des VM Private Cloud avec des appliances de sécurité déployées sur le Public Cloud.
+
+**Ajouter votre projet Public Cloud et votre bloc Additional IP au même vRack.**
+
+A des fins d'exemple pour ce guide, le bloc IP est `147.135.161.152/29`.<br>
+Nous utilisons la première IP utilisable `147.135.161.153` pour la première instance de SNS et nous utilisons temporairement la seconde IP utilisable `147.135.161.154` pour le second SNS.<br>
+L'adresse de la passerelle est `147.135.161.158`.
+
+Reportez-vous au guide « [Configurer un bloc IP dans un vRack](/pages/bare_metal_cloud/dedicated_servers/configuring-an-ip-block-in-a-vrack) » pour plus d'informations.
+
+Voici ci-dessous l'architecture que nous allons mettre en place.
+
+![SNS vrack](images/stormshield-ha-vrack.png){.thumbnail}
+
+#### Configurer le réseau OpenStack
+
+Créez le réseau privé pour les interfaces externes SNS :
+
+```bash
+openstack network create --provider-network-type vrack --provider-segment 0 --disable-port-security stormshield-ext
+```
+
+```bash
+openstack subnet create --network stormshield-ext --subnet-range 192.168.1.0/29 --dhcp stormshield-ext
+```
+
+Créez le réseau privé pour les interfaces internes du SNS :
+
+```bash
+openstack network create --provider-network-type vrack --provider-segment 200 --disable-port-security stormshield-vlan200
+```
+
+```bash
+openstack subnet create --network stormshield-vlan200 --subnet-range 10.200.0.0/16 --dhcp --dns-nameserver <dns_address_ip> stormshield-vlan200
+```
+
+Créez le réseau privé pour les interfaces SNS HA (**H**igh **A**vailability) :
+
+```bash
+openstack network create --provider-network-type vrack --provider-segment 199 --disable-port-security stormshield-ha
+```
+
+```bash
+openstack subnet create --network stormshield-ha --subnet-range 192.168.2.0/29 --dhcp --gateway none stormshield-ha
+```
+
+#### Déployer les instances SNS
+
+Rendez-vous dans la section `download` du [site officiel de Stormshield](https://documentation.stormshield.eu/SNS/v4/fr/Content/PAYG_Deployment_Guide/Downloading_installation_file.htm){.external}. Connectez-vous à votre compte Stormshield et suivez les instructions pour télécharger l'image Stormshield OpenStack.
+
+Rendez-vous dans le dossier où vous avez téléchargé votre image SNS Openstack et téléchargez l'image (pour ce tutoriel, nous utilisons l'image `utm-SNS-EVA-4.8.3-openstack.qcow2`) :
```

Comment:
```suggestion
Rendez-vous dans le dossier où vous avez téléchargé votre image SNS Openstack et téléversez l'image (pour ce tutoriel, nous utilisons l'image `utm-SNS-EVA-4.8.3-openstack.qcow2`) :
```
EN guide mentions upload, not download. Can you check please?

Reply:
Yes "téléversez" or "importez" is correct